src/styles/GlobalStyles.ts

```diff
@@ -452,6 +452,29 @@ textarea {
   padding: 0.2em 0;
 }
 
+h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6 {
```

Comment:
this looks wrong, no?

Reply:
I copied this from beta.quran.com actually. I'm not sure how you define "wrong". Mind to clarify?

Reply:
LEt's remove it. We won't copy everything over from beta